docs/overlays/README.md

```diff
@@ -48,13 +48,7 @@ import cities from 'example-cities';
 </MapGL>
 ```
 
-Want to create and share your own overlay? Fork the
-[react-map-gl-example-overlay](https://github.com/vicapow/react-map-gl-example-overlay)
-project to get started.
-
+Want to create and share your own overlay? Check the [examples/additional-overlays](https://github.com/uber/react-map-gl/tree/master/examples/additional-overlays) folder for examples.
 
 ## Remarks
-* **These overlays are currently not tested with perspective mode, although
-  they should in theory be compatible with perspective enabled viewports in
-  [viewport-mercator-project](https://github.com/uber-common/viewport-mercator-project)**
-* In v1, overlays were exported directly from 'react-map-gl'.
+* In `react-map-gl` v1, overlays were exported directly from 'react-map-gl'. Additional v1 examples can be found in the [react-map-gl-example-overlay](https://github.com/vicapow/react-map-gl-example-overlay) project.
```

Comment:
I know this is unrelated to the PR-ish, but do we still want to reference that repo. It hasn't been updated since Jan 2016 :p.